### ISSUE TITLE
img2img script generates corrupted images

### ISSUE THREAD

**Issue by araleza** [Alpha-VLLM/Lumina-T2X#98]
Hi, I've been trying to use your `sample_img2img.sh` script. But, it seems to generate corrupted output, even when given a Lumina-generated image as input.

For example, I can generate a nice image by calling the `sample_img2img.sh` script, with strength=1.0 (so it's basically just sample.sh):

'close up of woman, outdoors':
![image](https://github.com/Alpha-VLLM/Lumina-T2X/assets/70412719/fd2bad6f-2fc5-4063-928e-6aa7d16e8238)

but if I then use this image as the input, and change the strength to 0.15, I get this output:
![image](https://github.com/Alpha-VLLM/Lumina-T2X/assets/70412719/5b0f84d0-b2d2-473c-97aa-c3428f15406f)

Here are the parameters I'm using in sample_img2img.sh:
```
# Lumina-Next supports any resolution (up to 2K)
# res="1024:024x1024 1536:1536x1536 1664:1664x1664 1792:1792x1792 2048:2048x2048"
res=1024:1024x1024
t=4
cfg=4.0
seed=25
steps=50
solver=euler
strength=0.15
image_dir=input.png
model_dir=../../models/next/
cap_dir=caption.txt
out_dir=output_dir
```
I'm using the lumina_next_t2i_mini version.

**Comment by thelemuet** [comment]
Could it be that you are using the same seed during img2img generation that was used to generate the initial image? I could be way off since you seemed to imply this was also happening with init images not previously generated with Lumina, but suggesting just in case it helps.

**Comment by araleza** [comment]
@thelemuet, actually yes, you're right, changing the seed gets rid of the mess, and produces a high quality output image.

> you seemed to imply this was also happening with init images not previously generated with Lumina

Well spotted, the actual problem I had originally that made me create this issue is that I had a non-Lumina source image, and it was getting badly blurred when I did img2img with denoising strength 0.15. I wanted to create this GitHub issue about it, so I made these new images with Lumina. But like you spotted, I've introduced a second (likely unrelated) problem by using the same seed for both passes.

I'll go back to my original image and see if I can make a better test case. But I'll close this one in the meantime. Thanks for your help.
